client/src/components/VolunteersForm.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+
+class VolunteersForm extends React.Component {
+state={
+    firstName:"",
+    lastName:"",
+    email:"",
+    phoneNumber:"",
+    password:"",
```

Comment:
password should not be part of the form - they are not signing up for a secure account, they're just expressing their interest to volunteer.

Reply:
yes you are right
thanks